### ISSUE TITLE
CLI: Enable automated interactions with analyzed pages

### ISSUE THREAD

**Issue by amedina** [GoogleChromeLabs/ps-analysis-tool#204]
## Feature Request

**Description:**
CLI capabilities to exercise CUJs, maximizing the "testing surface", including cookies and storage APIs.

**Motivation:**
When running automated analyses, the CLI must ensure that basic interactions such as responding to GDPR prompts, are exercised. 

**Acceptance Criteria:**
- [x] GDPR banners are accepted. 

**Additional Information:**
[Include any additional information or context that may be helpful, such as related resources or examples.]

**Comment by kjnanda** [comment]
@amovar18 Please check my comment below. Listing down the website I am testing along with complete report.

| Websites QAed | isCookiePresent ? | Remark |  Screencap/Screenshot |
|---- |---- |---- |--- |
| https://www.aljazeera.com/ | ✅ | - |
| https://www.singaporeair.com/ | ✅  | Since it does not show GDPR banner in non headless mode, it won't show the cookie in CLI report  | - |
| https://www.airindia.com/ | ✅  | Since it does not show GDPR banner in non headless mode, it won't show the cookie in CLI report | - |
| https://www.roche.com/ | ❌ | **It shows GDPR banner in non headless mode but I don't see `OptanonAlertBoxClosed` cookie present in CLI report.** | ![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/15173772/12734b55-c87a-4bb7-990f-7157f8b70919)<br/>![image](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/15173772/dbb26d13-58e9-487b-8106-984ee38b0384) |


```
Test Environment:
1. Branch - feat/automate-cli
5. Chrome Browser : V120
6. OS : Mac V13.5.2
```

**Comment by maitreyie-chavan** [comment]
Addressed and released as a part of [v0.7](https://github.com/GoogleChromeLabs/ps-analysis-tool/releases/tag/v0.7.0)
